### ISSUE TITLE
npm install warning: "deprecated node-uuid@1.4.7: use uuid module instead"

### ISSUE THREAD

**Issue by FredKSchott** [Polymer/wct-local#54]
This warning is logged when installing WCT, Polymer CLI, and any other consumers.

I tracked it down to `launchpad@0.5.4` > `restify@4.3.0`

```
  └─┬ wct-local@2.0.14
    └─┬ launchpad@0.5.4
      └─┬ restify@4.3.0
        └── node-uuid@1.4.7 
```

Restify has only just fixed this in v5.0.0-beta-3.0. We should keep an eye out for their 5.0 release and then get launchpad to update.

**Comment by diddledan** [comment]
reference: daffl/launchpad#72

**Comment by FredKSchott** [comment]
@diddledan nice! Thanks for the update. We aren't using those server features so we should be fine to update. I'll try it out today to confirm.

**Comment by stramel** [comment]
@FredKSchott Closed with #57

**Comment by usergenic** [comment]
Looks at least superceded by #697 if not obsolete
